planner/core/point_get_plan.go

```diff
@@ -137,7 +141,7 @@ func (p *PointGetPlan) AccessObject() string {
 // OperatorInfo implements dataAccesser interface.
 func (p *PointGetPlan) OperatorInfo(normalized bool) string {
 	buffer := bytes.NewBufferString("")
-	if p.IndexInfo == nil {
+	if p.Handle != nil {
```

Comment:
Should `BatchPointGetPlan` be changed too?

Reply:
No. Clustered handles saved in index values.